app-frontend/src/app/components/common/search/search.js

```diff
@@ -0,0 +1,39 @@
+import angular from 'angular';
+import searchTpl from './search.html';
+
+export default angular.module('components.common.search', [])
+    .component('rfSearch', {
```

Comment:
I'd probably do it as 
```
const rfSearchComponent = {
...
};
class SearchController {
...
}

export default angular.module('components.common.search', [])
.component('rfSearch', rfSearchComponent)
.controller('SearchController', SearchController)
```
mostly it avoids the extra indents and defining classes inline feels weird.

Reply:
I like this approach better to avoid the unnecessary indents

Reply:
And so it shall be. :+1: